ee/candi/cloud-providers/dynamix/openapi/cluster_configuration.yaml

```diff
@@ -120,11 +120,6 @@ apiVersions:
                 description: |
                   External network ID.
                 example: 1
-              vinsNetworkId:
```

Comment:
Rename to `internalNetworkId`. We need a consistent naming across all providers. Users will be confused by the `vins` prefix.

Reply:
this parameter has been removed